src/App.tsx

```diff
@@ -1,7 +1,45 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+// export const App: React.FC = () => (
+//   <div className="App">
+//     <p className="App__message">The last pressed key is [Enter]</p>
+//   </div>
+// );
+
+type State = {
+  pressedKey: string | null;
+};
+
+type Props = {};
+
+export class App extends React.Component<Props, State> {
+  state: Readonly<State> = {
+    pressedKey: null,
+  };
+
+  UNSAFE_componentWillMount(): void {
```

Comment:
you can use `componentWillUnmount` here
```suggestion
  componentWillUnmount(): void {
```